walkthrough.ipynb

```diff
@@ -205,16 +205,13 @@
     "<img src=\"openapi.png\" alt=\"drawing\" width=\"200\"/>\n",
     "\n",
     "[Here](https://lumigator.mzai.dev/docs) are the docs for the Lumigator API. In looking at them, we can see that we have 7 endpoints. \n",
-    "The application is split up into `jobs`, `deployments`, `datasets`, `experiments`, and `completions`.\n",
+    "The application is split up into:\n",
```

Comment:
I changed this to mirror the endpoint names and order but LMK if that sounds good to you!